### ISSUE TITLE
.NET Standard support

### ISSUE THREAD

**Issue by bothzoli** [sfmohassel/libc.eventbus#7]
I just found your library and it would be a perfect match for me, but alas I work with .NET 4.8.
Would you consider adding .NET Standard support as well (and if there also adding .NET 4.8, .NET 8 and remove unsupported versions)?
I made the changes locally and it works without issues.
The only change needed is to update the target frameworks as such, but I'm happy to create a PR for it:

```xml
<TargetFrameworks>net48;netstandard2.0;net6.0;net8.0</TargetFrameworks>
```

**Comment by sfmohassel** [comment]
I see no issues with thiss

**Comment by bothzoli** [comment]
Awesome, thanks.
Would you like me to create the PR? I'm more than happy to.

**Comment by sfmohassel** [comment]
Yes, by all means :)

**Comment by bothzoli** [comment]
Hey @sfmohassel,
I just created #8.
I was not sure about the version bump. Since there are no real changes, I would've gone with a patch bump.
However, the support of .NET 7 is dropped (as it's no longer officially supported) which could be considered a breaking change and hence a major bump.
But that also didn't feel quite right, so I went with a minor version bump.
Feel free to change it as you see fit :)
